src/components/SidePanes/CatalogInclusionPane.test.jsx

```diff
@@ -22,6 +22,21 @@ describe('CatalogInclusionPane', () => {
     expect(toggle.props().checked).toBe(false);
   });
 
+  it('allow course runs who have been reviewed', () => {
+    const wrapper = mount(<CatalogInclusionPane
+      courseUuid={mockUuid}
+      subInclusion={mockInclusion}
+      draftStatuses={['reviewed']}
+      orgInclusion
+    />);
+    wrapper.find(CatalogInclusionPane);
+    const title = wrapper.find('Enterprise Subscriptions');
+    expect(title);
+    const toggle = wrapper.find('.pgn__form-switch-input');
+    toggle.simulate('change', { target: { checked: false } });
```

Comment:
`toggle.simulate('change', { target: { checked: false } });`

I believe it should be `true` here because the default value is `false`